wheel_radius: 0.0325
base_name: "base_wheel"
wheel_name: "right_wheel"
right_wheel: false